suite-common/wallet-utils/src/transactionUtils.ts

```diff
@@ -476,7 +476,9 @@ export const getTxOperation = (
 };
 
 export const isNftTokenTransfer = (transfer: TokenTransfer) =>
-    ['ERC1155', 'ERC721'].includes(transfer.standard || '');
+    ['ERC1155', 'ERC721'].includes(transfer?.standard || '');
+
+export const isMultitoken = (transfer: TokenTransfer) => !!transfer?.multiTokenValues;
```

Comment:
```suggestion
export const IsNftMultitokenTransfer = (transfer: TokenTransfer) => !!transfer?.multiTokenValues;
```

and btw it return `true` when multitokens array is defined but empty

Reply:
I'm not sure it's a problem in this case, can this even be a case? I mean can a transfer have multitoken value defined but have transaction amount outside of it? I'm not so sure, will have to test it out

Reply:
I have no idea but we should be ready for weird cases :D

Reply:
I added a check for length there just in case, I don't think it's possible, maybe only in a very weird nonsense contract